api/agent/grpc/runner.proto

```diff
@@ -100,6 +100,11 @@ message LogRequestMsg {
 
     // Acknowledgement and re-enable stream message sent by Collectors to Runners
     message Ack {
+        enum Type {
+            PUSH = 0;
```

Comment:
Minor, but what if these were called "received" and "committed" or something, just to make it a little more obvious what they mean?